src/providers/reversing-distances/__tests__/reversing-distances.spec.ts

```diff
@@ -18,18 +17,27 @@ describe('ReversingDistancesProvider', () => {
   });
 
   describe('getDistanceLength', () => {
-    // TODO: Use CAT C type
-    const vehicleDetails: CatBEUniqueTypes.VehicleDetails = {
+    const vehicleDetails: CatCVehicleDetailsUnion = {
       vehicleLength: 15,
       vehicleWidth: 2,
     };
 
+    const longVehicleDetails: CatCVehicleDetailsUnion = {
+      vehicleLength: 20,
+      vehicleWidth: 2,
+    };
+
     describe('Category C', () => {
       it('should return a start distance value 3 and a half times the vehicle length', () => {
         const result = reversingDistancesProvider.getDistanceLength(vehicleDetails, TestCategory.C);
         expect(result.startDistance).toEqual(52.5);
       });
 
+      it('should return a value 3 and a half times vehicle length is greater then 16.5', () => {
```

Comment:
This description doesn't quite read right...
Maybe 'should return a value 3 and a half times vehicle length **if** greater **than** 16.5' ?